deploy/operator/pmem-csi-operator.yaml

```diff
@@ -140,6 +140,20 @@ subjects:
   name: pmem-csi-operator
   namespace: pmem-csi
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pmem-csi-operator-metrics
+  namespace: pmem-csi
+spec:
+  ports:
+  - port: 8080
+    targetPort: 8080
+  selector:
+    app: pmem-csi-operator
+    name: pmem-csi-operator
+  type: NodePort
+---
```

Comment:
We cannot assume that the 8080 node port is free. Why do you need a node port?

Reply:
> Why do you need a node port?

Yes, it shouldn't be. I took this from the driver metrics service which is using NodePort. I will fix this.

Reply:
Done.